src/routers/GetActiveMatchRouter.ts

```diff
@@ -0,0 +1,23 @@
+import { Request } from 'express';
+import { SerializedMatch } from '../common/types';
+import AuthenticatedApplicationRouter from '../utils/AuthenticatedApplicationRouter';
+import MatchRepo from '../repos/MatchRepo';
+
+class GetActiveMatchRouter extends AuthenticatedApplicationRouter<SerializedMatch[]> {
```

Comment:
This should be `GetWeeklyMatch` to be consistent with the naming change proposed.